shared/middleware/animation_library_api.rb

```diff
@@ -110,6 +110,38 @@ class AnimationLibraryApi < Sinatra::Base
     not_found
   end
 
+  #
+  # GET /api/v1/animation-library/default_spritelab/
+  #
+  # Retrieve the default sprite list from S3
+  get %r{/api/v1/animation-library/default_spritelab} do
+    result = Aws::S3::Bucket.
+      new(ANIMATION_LIBRARY_BUCKET, client: AWS::S3.create_client).
+      object("animation-manifests/manifests_levelbuilder/defaults.json").
```

Comment:
nit: pull this string into a constant like `ANIMATION_LIBRARY_BUCKET` at the top of this file